tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/valid_death_dates_test.py

```diff
@@ -89,8 +119,16 @@ def setUp(self):
         fq_dataset_name = self.fq_table_names[0].split('.')
         self.fq_dataset_name = '.'.join(fq_dataset_name[:-1])
 
+        self.copy_vocab_tables()
+
         super().setUp()
 
+    def copy_vocab_tables(self):
```

Comment:
nit:  This is an optional thing, but I'm curious about why you are creating a function to run a loop instead of just running the loop.

Reply:
I had copied this method from Nishanth, so you would have to ask him. But if I had to guess, it would be because it's more portable.

Reply:
I would suggest that this isn't portable because you're not importing this function and using it, you're redefining it.  I would recommend just using the loop and adding a comment above it about what it's doing.  This is adding a bit of extra complexity for just 2 lines of logical code.

Reply:
Sounds good, will do that